code/game/objects/items/stacks/tiles/tile_iron.dm

```diff
@@ -121,13 +197,73 @@
 	merge_type = /obj/item/stack/tile/iron/cafeteria
 	tile_rotate_dirs = list(SOUTH, NORTH)
 
+/obj/item/stack/tile/iron/white/textured
+	name = "white textured tile"
+	singular_name = "white textured floor tile"
+	icon_state = "tile_textured_white"
+	turf_type = /turf/open/floor/iron/white/textured
+	merge_type = /obj/item/stack/tile/iron/white/textured
+
+/obj/item/stack/tile/iron/white/textured_edge
+	name = "white textured edge tile"
+	singular_name = "white textured edge floor tile"
+	icon_state = "tile_textured_white_edge"
+	turf_type = /turf/open/floor/iron/white/textured_edge
+	merge_type = /obj/item/stack/tile/iron/white/textured_edge
+	tile_rotate_dirs = list(SOUTH, NORTH, EAST, WEST)
+
+/obj/item/stack/tile/iron/white/textured_large1
+	name = "large white textured tile"
+	singular_name = "large white textured floor tile"
+	icon_state = "tile_textured_white_large1"
+	turf_type = /turf/open/floor/iron/white/textured_large1
+	merge_type = /obj/item/stack/tile/iron/white/textured_large1
+	tile_rotate_dirs = list(SOUTH, NORTH, EAST, WEST)
+
+/obj/item/stack/tile/iron/white/textured_large2
+	name = "larger white textured tile"
+	singular_name = "larger white textured floor tile"
+	icon_state = "tile_textured_white_large2"
+	turf_type = /turf/open/floor/iron/white/textured_large2
+	merge_type = /obj/item/stack/tile/iron/white/textured_large2
+
 /obj/item/stack/tile/iron/recharge_floor
 	name = "recharge floor tile"
 	singular_name = "recharge floor tile"
 	icon_state = "tile_recharge"
 	turf_type = /turf/open/floor/iron/recharge_floor
 	merge_type = /obj/item/stack/tile/iron/recharge_floor
 
+/obj/item/stack/tile/iron/smooth
+	name = "smooth tile"
+	singular_name = "smooth floor tile"
+	icon_state = "tile_smooth"
+	turf_type = /turf/open/floor/iron/smooth
+	merge_type = /obj/item/stack/tile/iron/smooth
+
+/obj/item/stack/tile/iron/smooth_edge
+	name = "smooth edge tile"
+	singular_name = "smooth edge floor tile"
+	icon_state = "tile_smooth_edge"
+	turf_type = /turf/open/floor/iron/smooth_edge
+	merge_type = /obj/item/stack/tile/iron/smooth_edge
+	tile_rotate_dirs = list(SOUTH, NORTH, EAST, WEST)
+
+/obj/item/stack/tile/iron/smooth_large1
+	name = "large smooth tile"
+	singular_name = "large smooth floor tile"
+	icon_state = "tile_smooth_large1"
+	turf_type = /turf/open/floor/iron/smooth_large1
+	merge_type = /obj/item/stack/tile/iron/smooth_large1
+	tile_rotate_dirs = list(SOUTH, NORTH, EAST, WEST)
```

Comment:
why 4 dirs if it only has 2 dirs that change the sprite?

Reply:
cause im idiot